server/index.js

```diff
@@ -16,9 +16,9 @@ app.use('/', expressStaticGzip(path.join(__dirname, '/../public'), {
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
-app.use(function(req, res, next) {
-  res.header("Access-Control-Allow-Origin", "*");
-  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+app.use((req, res, next) => {
```

Comment:
Looks good! This might help clean up your Express code for cors requests though...
https://www.npmjs.com/package/cors